### DISCUSSION TITLE
How to migrate my Netbox to a new machine ?

### DISCUSSION THREAD

**Discussion by Jason-MK** [netbox-community/netbox#7192] [Q&A] [answered]
Hello ,

I want ask how to move my Netbox machine to another one ? I mean I have my Netbox on a Debian machine and I want to create a new machine and move all the information to the new one ? how can I do that ? 

Best Regards

**Comment by candlerb** [comment, marked as answer]
The process is documented here: https://netbox.readthedocs.io/en/stable/administration/replicating-netbox/

If the target machine has a newer version of Netbox then remember to run `./upgrade.sh` on the new machine after importing the postgres dump.
